duckduckgo/iOS
# Display alert when editing a login which is a duplicate of another

Task/Issue URL: https://app.asana.com/0/1201462886803403/1203867198849545
Tech Design URL:
CC:

**Description**:
Present a duplicate login alert to the user if they update an existing login to match another existing one 



**Steps to test this PR**:
1. Create 2 logins with very similar credentials for the same site e.g. 
- Username = `Dax`, Password = `daxtheduck`, Website = `fill.dev`
- Username = `DaxDuck`, Password = `daxtheduck`, Website = `fill.dev`
2. Modify the second login username to be the same as the first one (`DaxDuck` updates to `Dax`)
3. Confirm a duplicate login alert is presented 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
